ci-operator/config/openshift/moactl/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
+- vkareh
+- jharrington22
+- jeremyeder
+options: {}
+approvers:
```

Comment:
Just curious, should this one be `reviewers`?

Reply:
Yeah it should, or not be here at all :)